pkg/network/network.go

```diff
@@ -155,6 +185,22 @@ func NewConfigFromConfigMap(configMap *corev1.ConfigMap) (*Config, error) {
 		nc.DomainTemplate = dt
 	}
 
+	nc.AutoTLS = strings.ToLower(configMap.Data[AutoTLSKey]) == "enabled"
+
+	switch strings.ToLower(configMap.Data[HTTPProtocolKey]) {
```

Comment:
If you use Title-case instead of Lower-case as the normalized case then you don't have to do ToLower on all of your constants.

https://golang.org/pkg/strings/#ToTitle

Reply:
I changed to use Lower-case as the normalized case to get rid of some ToLower functions.
With lower-case normalization, we can map `Enabled`, `enabled`, and `ENABLED` to HTTPEnabled variable.